model/job.go

```diff
@@ -230,20 +274,43 @@ func (j *Job) MergeSpec(otherJob *Job) {
 	j.Properties = append(j.Properties, otherJob.Properties...)
 }
 
+type configConsumes struct {
+	Role string `json:"role"`
+	Job  string `json:"job"`
+}
+
 // WriteConfigs merges the job's spec with the opinions and returns the result as JSON.
 func (j *Job) WriteConfigs(role *Role, lightOpinionsPath, darkOpinionsPath string) ([]byte, error) {
-	config, err := initializeConfigJSON()
-	if err != nil {
-		return nil, err
-	}
-
-	config["job"].(map[string]interface{})["name"] = role.Name
+	var config struct {
+		Job struct {
+			Name      string `json:"name"`
+			Templates []struct {
```

Comment:
Oops, missed this comment. I'll add something for this.
This isn't a `JobTemplate` because its' just being used to serialize to JSON… though I should think about it.